kubernetes/resource_kubernetes_env_test.go

```diff
@@ -242,6 +420,77 @@ func createCronJobEnv(t *testing.T, name, namespace string) error {
 	return err
 }
 
+func createCronJobInitContainerEnv(t *testing.T, name, namespace string) error {
+	conn, err := testAccProvider.Meta().(kubeClientsets).MainClientset()
+	if err != nil {
+		return err
+	}
+	ctx := context.Background()
+
+	var failJobLimit int32 = 5
+	var startingDeadlineSeconds int64 = 2
+	var successfulJobsLimit int32 = 2
+	var boLimit int32 = 2
+	var ttl int32 = 2
```

Comment:
Just fyi there's a util package [k8s.io/utils/pointer](https://pkg.go.dev/k8s.io/utils/pointer) that you can use to more conveniently set these fields that are pointers.